codes/codes.go

```diff
@@ -19,9 +19,8 @@
 // Package codes defines the canonical error codes used by gRPC. It is
 // consistent across various languages.
 package codes // import "google.golang.org/grpc/codes"
-import (
-	"fmt"
-)
+
+import "fmt"
```

Comment:
Golang [tour](https://tour.golang.org/basics/2) suggests to use factored import statements. So we should change this statement back.

Reply:
OK, thank you!